Dockerfile

```diff
@@ -27,9 +27,6 @@ RUN yarn install --production --ignore-scripts --prefer-offline
 # E2E image for running tests (same as prod but without certs)
 FROM node:14.19.3-slim AS e2e
 
-RUN apt-get update \
-  && apt-get -y --no-install-recommends install openssl libc6
```

Comment:
@esacteksab is this expected to have this PR remove all of this? or am i missing something? just don't want to delete anything important..

Reply:
That was definitely a miss for me. I put it back. Thank you for catching this.